plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/CommitInfoEntry.java

```diff
@@ -34,7 +34,7 @@
     private final String clusterId;
     private final long readVersion;
     private final String isolationLevel;
-    private final boolean isBlindAppend;
+    private final Boolean isBlindAppend;
```

Comment:
Although this information is not currently used in Trino, there is some extra info that could be added here:
- operationMetrics
- userMetadata
- engineInfo

Also `operationParameters` seems to be a bit more complex than only a map of `<string, string> entries.

Reply:
Why the capital B Boolean? If it may not be present, use `Optional<Boolean>`